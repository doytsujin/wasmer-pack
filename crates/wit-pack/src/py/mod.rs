use anyhow::Error;

use crate::{Files, Interface, Metadata, Module};

/// Generate Python bindings.
pub fn generate_python(
    _metadata: &Metadata,
    _module: &Module,
    _interface: &Interface,
) -> Result<Files, Error> {
    todo!()
}
