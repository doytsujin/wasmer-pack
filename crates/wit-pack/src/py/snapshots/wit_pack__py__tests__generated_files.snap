---
source: crates/wit-pack/src/py/mod.rs
expression: "files[\"pyproject.toml\"].utf8_contents().unwrap()"
---
[project]
name = "wit_pack"
version = "1.2.3"
keywords = []
dependencies = ["wasmer", "wasmer_compiler_cranelift"]

